src/providers/VoiceFocusProvider/docs/useVoiceFocus.stories.mdx

```diff
@@ -5,7 +5,8 @@ import { useVoiceFocus } from '../';
 
 # useVoiceFocus
 
-The `useVoiceFocus` hook returns a function transforming a normal device to an Amazon Voice Focus device, and whether Amazon Voice Focus is supported.
+The `useVoiceFocus` hook returns a function transforming a normal audio device to an Amazon Voice Focus device, and whether Amazon Voice Focus is supported.
+Amazon Voice Focus related logs can be found in the dev tool when the feature is enabled.
```

Comment:
Could you please update "in the browser developer tools" like the `VoiceFocusProvider`?

Reply:
Sure, updated